src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1935,7 +1960,7 @@ public function testTrustedProxiesForwarded()
     {
         $request = Request::create('http://example.com/');
         $request->server->set('REMOTE_ADDR', '3.3.3.3');
-        $request->headers->set('FORWARDED', 'for=1.1.1.1, host=foo.example.com:8080, proto=https, for=2.2.2.2, host=real.example.com:8080');
```

Comment:
maybe we should keep this line, because it's going to work as a fallback, isn't it?